package-configuration/linux-cli/test_cli_install.py

```diff
@@ -31,4 +37,16 @@ def setUp(self):
         self.get_cloudify = get_cloudify
 
     def test_full_cli_install(self):
-        self.run_get_cloudify('-f -v -e=/tmp/temp_env/ --nosudo')
+        tempdir = tempfile.mkdtemp()
+        self.run_get_cloudify('-f -v -e={0} --nosudo'.format(tempdir))
+        p = self.get_cloudify.run('{0}/bin/cfy --version'.format(tempdir))
+        self.assertIn('Cloudify CLI 3', p[2])
+        shutil.rmtree(tempdir)
+
+    def test_install_from_source(self):
+        tempdir = tempfile.mkdtemp()
+        self.run_get_cloudify('-s {0} -v -e={1}'.format(
+            cloudify_cli_url, tempdir))
+        p = self.get_cloudify.run('{0}/bin/cfy --version'.format(tempdir))
+        self.assertIn('Cloudify CLI 3', p[2])
+        shutil.rmtree(tempdir)
```

Comment:
actually..  the right way to do this is use a cleanup method in the class which is builtl-in to testtools or unittest.. but try finally is a good alternative :)